IntroOOP/Singleton/src/com/globant/singletondb/DBConnection.java

```diff
@@ -0,0 +1,36 @@
+package com.globant.singletondb;
+import java.sql.Connection;
+import java.sql.DriverManager;
+
```

Comment:
Ponerle comentarios (JavaDocs) a la clase

Reply:
Hello Leandro, I already did on [this commit]https://github.com/java-bootcamp-rcia/facundo-bohle/pull/1/commits/e59083731957e0c3f9ffeb0d22d0f9a02f757b45) in the [topic-introOOP branch](https://github.com/bohledevs/facundo-bohle/tree/topic-introOOP) . 

This is the topic-Maven branch, because of an incorrect merge those changes are shown here, but they shoudn't. My bad.